docs/assets/scss/header.scss

```diff
@@ -22,16 +22,6 @@
   }
 
   @media screen and (max-width: 991.98px) {
-    .td-navbar-nav-scroll {
-      height: auto;
-    }
-    .navbar-nav {
-      padding-bottom: 0;
-      overflow: hidden;
-    }
-  }
-
-  @media screen and (max-width: 767px) {
```

Comment:
curious: why did we needed to remove this media size and change the other from 767 to 991.98px?

Reply:
Hi! Sorry for the delayed response. The number of menu items is too large to fit in a single line, so I decided to switch to the mobile version earlier to prevent the last button from constantly getting cut off at the edge of the screen

<img width="767" alt="image" src="https://github.com/user-attachments/assets/08ab87b3-2490-4092-b7b9-941a9e5e4d7e" />

<img width="926" alt="image" src="https://github.com/user-attachments/assets/b1bc5ce5-0a18-4888-96e6-aec8b3c3cc0e" />